### PR TITLE
Fix broken postgrad teacher training link

### PR DESCRIPTION
**Pull request by ethax-ross**
The postgraduate teacher training service has changed domain; there used to be a redirect in place but that was recently removed so we need to point directly to the new service.

The IOP domain does not have a valid SSL certificate at the moment, so ignoring.

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
Thanks @ethax-ross 

FYI @emilytraff has [an open PR](https://github.com/DFE-Digital/get-into-teaching-app/pull/1788) on this page to change the link to Apply on 12 Oct - will this need to be rebased when you've merged your change?

File: app/views/content/blog/how-to-apply-for-teacher-training.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Our free events are the perfect opportunity to find out more about teacher train
 
 ### 1. Find teacher training courses
 
-Use the Department for Education’s ‘[find postgraduate teacher training courses](https://find-postgraduate-teacher-training.education.gov.uk/)’ to search for courses in England. You’ll be able to search by location, training provider and subject. When you apply, you have three choices for teacher training providers. It’s a good idea to select three choices; the more flexible you are, the more likely it is that you’ll secure a place.
+Use the Department for Education’s ‘[find postgraduate teacher training courses](https://www.find-postgraduate-teacher-training.service.gov.uk/)’ to search for courses in England. You’ll be able to search by location, training provider and subject. When you apply, you have three choices for teacher training providers. It’s a good idea to select three choices; the more flexible you are, the more likely it is that you’ll secure a place.
```
Comment (gemmadallmandfe) [inline app/views/content/blog/how-to-apply-for-teacher-training.md:57]:
```suggestion
Use the Department for Education’s [find postgraduate teacher training courses](https://www.find-postgraduate-teacher-training.service.gov.uk/) service to search for courses in England. You’ll be able to search by location, training provider and subject. When you apply, you have three choices for teacher training providers. It’s a good idea to select three choices; the more flexible you are, the more likely it is that you’ll secure a place.
```

**Comment by ethax-ross** [comment]
@gemmadallmandfe it shouldn't be a conflict as we're changing different parts of the file, but if it does I can rebase it for her

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**